robosuite/utils/control_utils.py

```diff
@@ -0,0 +1,167 @@
+import numpy as np
+import robosuite.utils.transform_utils as trans
+import scipy.linalg
+
+
+def nullspace_torques(mass_matrix, nullspace_matrix, initial_joint, joint_pos, joint_vel):
+    joint_kp = 10
+    joint_kv = np.sqrt(joint_kp) * 2
```

Comment:
comment that this corresponds to critical damping

Reply:
@cremebrule following up on this

Reply:
done! added line by line comments for this function as well